File: src/main/java/project/rpg/player/job/thing/Adventurer.java
package project.rpg.player.job.thing;

import org.bukkit.entity.Player;
import project.rpg.player.job.JobBase;
import project.rpg.player.job.Jobs;

public class Adventurer extends JobBase {

    @Override
    public void reload() {
        //추가 예정
    }

    @Override
    public void setNextJobs() {
        this._nextJobs.add(Jobs.SWORDSMAN);
        this._nextJobs.add(Jobs.WARRIOR);
        this._nextJobs.add(Jobs.THIEF);
        //궁수
        //이류무사 dlc
    }

    @Override
    public void setJobSkills() {
        //추가 예정
    }

    public Adventurer(Player player) {
        super(Jobs.ADVENTURER,  player);
    }

}

File: src/main/java/project/rpg/player/job/thing/Adventurer.kt
package project.rpg.player.job.thing

import org.bukkit.entity.Player
import project.rpg.player.job.JobBase
import project.rpg.player.job.JobType

class Adventurer(player: Player) : JobBase(player, JobType.ADVENTURER ) {
    override fun reload() {
        //추가 예정
    }

    override fun setNextJobs() {
        nextJobs.add(JobType.SWORDSMAN)
        nextJobs.add(JobType.WARRIOR)
        nextJobs.add(JobType.THIEF)
        //궁수
        //이류무사 dlc
    }

    override fun setJobSkills() {
        //추가 예정
    }
}